BUILD.gn

```diff
@@ -8,6 +8,9 @@ import("//build_extra/rust/rust.gni")
 config("deno_config") {
   include_dirs = [ "third_party/v8" ]  # This allows us to v8/src/base/ libraries.
   configs = [ "third_party/v8:external_config" ]
+  if (is_debug) {
+    defines = [ "DEBUG" ]
+  }
```

Comment:
For the DCHECK below

Reply:
Surprising that DEBUG wasn't already set...